xml/admin_ceph_upgrade.xml

```diff
@@ -689,6 +686,14 @@ exported keyring for client.bootstrap-osd
  "flags": "sortbitwise,recovery_deletes,purged_snapdirs",</screen>
     </tip>
    </step>
+   <step>
+    <para>
+     Upgrade the &sadmin; node to &sls; 12 SP3 and &storage; 5. If you use
+     SCC-registered systems, we recommend <command>zypper migration</command>.
+     After the upgrade, ensure that only repositories for &sls; 12 SP3 and
+     &storage; 5 are active (and refreshed) on the &sadmin; node before proceeding.
```

Comment:
We have no sadmin entity. You can either add it to the entity-decl.ent or replace \&sadmin; with \&salt; master.